src/rosdistro/freeze_source.py

```diff
@@ -84,7 +84,7 @@ def freeze_distribution_sources(dist, release_version=False, release_tag=False,
 def _get_repo_info(url, retry=2, retry_period=1):
     cmd = ['git', 'ls-remote', url]
     try:
-        return subprocess.check_output(cmd).splitlines()
+        return subprocess.check_output(cmd).decode().splitlines()
```

Comment:
The returned value here was previously a list of strings. Now it is on Python 2 a list of unicodes, and on Python 3 a list of str.